modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/js/components/ScriptManagementContainer.tsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayAlert from '@clayui/alert';
+import ClayButton from '@clayui/button';
+import {Text} from '@clayui/core';
+import {ClayCheckbox} from '@clayui/form';
+import {createResourceURL, fetch, openToast} from 'frontend-js-web';
+import React, {useState} from 'react';
+
+import './ScriptManagementContainer.scss';
+
+interface ScriptManagementContainerProps {
+	allowScriptContentBeExecutedOrIncluded: boolean;
+	baseResourceURL: string;
+}
+
+export default function ScriptManagementContainer({
+	allowScriptContentBeExecutedOrIncluded,
+	baseResourceURL,
+}: ScriptManagementContainerProps) {
+	const [allowScriptContent, setAllowScriptContent] = useState(
+		allowScriptContentBeExecutedOrIncluded
+	);
+
+	const handleSaveSystemConfiguration = async () => {
+		const response = await fetch(
+			createResourceURL(baseResourceURL, {
+				allowScriptContentBeExecutedOrIncluded: allowScriptContent,
+				p_p_resource_id:
+					'/system_settings/edit_script_management_configuration',
+			}).toString()
+		);
+
+		openToast({
+			message: response.ok
+				? Liferay.Language.get('your-request-completed-successfully')
+				: Liferay.Language.get('an-error-occurred'),
+			type: response.ok ? 'success' : 'danger',
+		});
+	};
+
+	return (
+		<div className="lfr__script-management-container">
+			<Text as="span" size={7} weight="bolder">
+				{Liferay.Language.get('script-management')}
+			</Text>
+
+			<ClayAlert displayType="info" title="Info:">
+				{Liferay.Language.get(
+					'this-configuration-is-not-saved-yet.-the-values-shown-are-the-default'
+				)}
+			</ClayAlert>
+
+			<div className="lfr__script-management-checkbox-container">
+				<div className="lfr__script-management-checkbox-label-container">
+					<ClayCheckbox
+						aria-label="Option 1"
+						checked={allowScriptContent}
+						onChange={() =>
+							setAllowScriptContent(!allowScriptContent)
+						}
+					/>
+
+					<Text as="span" size={4} weight="semi-bold">
+						{Liferay.Language.get(
+							'allow-instance-admin-to-create-and-execute-code-in-liferay-dxp'
+						)}
+					</Text>
+				</div>
+
+				<Text as="span" color="secondary" size={4} weight="normal">
```

Comment:
Hi @danvb112, `as="span"`, `size={4}` and `Weight="normal"` are default values ​​for `ClayText`. So, do you think they are necessary here?

If you think that the default values ​​are not necessary, please check the other uses and remove if necessary.

Reply:
Hey @igor-franca , thanks for noticing this, I didn't pay attention to the default values ​​of the Text component, I removed the unnecessary ones